user_sync/config.py

```diff
@@ -386,6 +387,28 @@ def get_directory_extension_options(self):
                         raise AssertionError("No after_mapping_hook found in extension configuration")
         return options
 
+    def get_post_sync_options(self):
+        """
+        Read the post_sync options from main_config_file, if there are any modules specified, and return its dictionary of options
+        :return: dict
+        """
+
+        ps_opts = self.main_config.get_dict_config('post_sync', True)
+        if not ps_opts:
+            return
+
+        connectors = ps_opts.get_dict('connectors')
+        module_list = ps_opts.get_list('modules')
+
+        try:
+            post_sync_modules = {m: self.get_dict_from_sources([connectors[m]]) for m in module_list}
+        except KeyError as e:
+            raise AssertionException("Error! Post-sync module " + str(e) + " specified without a configuration file...")
+
+        return {
+            'modules': post_sync_modules,
+        }
+
```

Comment:
We need extra validation added here to prevent confusing errors.  See https://github.com/adobe-dmeservices/user-sync.py/pull/93

Reply:
Cool, go ahead and merge it and I'll do some testing.

Reply:
Looks good. I made one small tweak so we don't import a private member from the post sync connectors module.